### DISCUSSION TITLE
OpenRC unit export option

### DISCUSSION THREAD

**Discussion by dimon222** [k9withabone/podlet#93] [Ideas]
Looking for non-systemd based alternative units for export. I know that systemd generation got transitioned into quadlets concept, and then this project appeared. But now it's unsolved task because previously you could generate systemd unit and then logically convert it to OpenRC unit with existing software, but now it's quadlet which doesn't have any available options to switch to OpenRC backend.
I realize that alternative path is to ask for OpenRC quadlet support in podman project, but it sounds to me less realistic/much bigger change relative to bunch of scripts in podlet to spit out non-quadlet units.

**Comment by k9withabone** [comment]
Sorry for not responding to this sooner. Could you not run Quadlet manually to create the systemd units, and then use the software you used before to create the OpenRC units? Quadlet is installed at `/usr/lib/systemd/system-generators/podman-system-generator`. You can control the input directory with the `QUADLET_UNIT_DIRS` environment variable, and you give the output directory as an argument.

**Reply by necrose99** [reply]
goose121/initify
Github 

Units to openrc ..
